docs/TESTS.md

```diff
@@ -1,15 +1,43 @@
 # Tests
 
-<!-- TODO: write document
+If you've downloaded an exercise with the CLI tool, you should see a Uiua file named after the exercise and a `tests.ua` file.
+This file contains all the test cases, which match those run by the test runner when you upload your solution.
+Feel free to add additional test cases for experimenting.
+However, modifying tests does not affect the online test-runner process.
 
-  This document should describe everything related to running tests in the track.
+## Local Testing
 
-  If your track uses skipped tests, this document can explain why thet is used and
-  how to unskip tests.
+After installing Uiua, run tests with `uiua test [PATH]`.
+To execute the test file, use `uiua tests tests.ua` in the terminal.
 
-  This document can also link to the testing framework documentation.
+For example, the output of an unedited `hello-world` exercise might look like this:
 
-  The contents of this document are displayed on the track's documentation
-  page at `https://exercism.org/docs/tracks/<track>/tests`.
+```bash
+$ uiua test tests.ua
+Error: Goodbye, Mars!
+  at tests.ua:3:1
+3 | ⍤⤙≍ "Hello, World!" Hello
+    ─
+Test failed
+```
 
-  See https://exercism.org/docs/building/tracks/docs for more information. -->
+The `Error` line displays your code's actual output.
+The line number indicates the location of the expected result in `tests.ua`.
+A correct solution produces:
+
+```bash
+$ uiua test tests.ua
+Test passed
+```
+
+## Skipped Tests
+
+Some exercises might include skipped tests to help you focus on early functionality before handling complex cases.
+Other exercises might exclude some test cases, the creator did not seem necessary to get a passing grade for the core of the concept being taught.
+To unskip tests, remove the relevant sections in the tests.ua file.
```

Comment:
We don't have this, do we?

Reply:
Not yet, but it might help with questions down the road. Should I drop that and re-insert it if we ever make such a thing?

Reply:
I think so